package.json

```diff
@@ -2,20 +2,27 @@
   "name": "quickstart-backbone-app",
   "private": true,
   "version": "0.0.0",
-  "dependencies": {},
+  "dependencies": {
+    "backbone": "^1.3.3",
+    "bootstrap": "^3.3.6",
+    "fh-js-sdk": "^2.14.4",
+    "jquery": "^3.0.0",
+    "underscore": "^1.8.3"
+  },
   "devDependencies": {
```

Comment:
I think you need `"browserify": "^13.0.1"` and  `"browserify-shim": "^3.8.12"` too